airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import unicode_literals
+
+from airflow.providers.amazon.aws.hooks.glue import AwsGlueJobHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class AWSGlueJobOperator(BaseOperator):
+    """
+    Creates an AWS Glue Job. AWS Glue is a serverless Spark
+    ETL service for running Spark Jobs on the AWS cloud.
+    Language support: Python and Scala
+
+    :param job_name: unique job name per AWS Account
+    :type str
```

Comment:
```suggestion
    :type job_name: Optional[str]
```

The same I mentioned in the AwsGlueHook ;)

Reply:
It would also be nice to the user if you can update documentation and add a note that if it is None it will be created...